tests/pubsub/testgossipsub.nim

```diff
@@ -360,12 +360,17 @@ suite "GossipSub":
           inc observed
       )
       obs2 = PubSubObserver(
+        onRecvAndValidated: proc(peer: PubSubPeer, msg: Message, msgId: MessageId) =
```

Comment:
Could this be added to the existing observer?

Reply:
Yes, definitely! But I guess it depends if we keep testing it here (and I add it to the existing observer) or if it will go on a separate test. LMK